### PR TITLE
Build tool migration

### PR DESCRIPTION
**Pull request by TimoKramer**
@whilo Feel free to change this PR according to your needs. Pretty old stuff in here but I don't know what is good and what is not. Maybe @groundedSAGE can help.

### CODE REVIEW

**Comment by whilo** [comment]
@TimoKramer Do the cljs tests run as well in CI/CD? This looks good to me overall.

**Comment by TimoKramer** [comment]
I took cljs tests out because they were failing. You can run them with `./bin/run-cljstests`. You'll need chrome installed.

**Comment by TimoKramer** [comment]
I could also take out all the changes to the sources and only apply the ci. This was basically a very long standing PR opened by @groundedSAGE

**Comment by whilo** [comment]
@groundedSAGE Can you take a look into (re)activating the cljs tests? This is kind of important if we want to keep cljs support around.

File: test/superv/async_test.cljc [hunk 1]
```diff
@@ -335,92 +341,92 @@
                   42))))
      (let [start-fn (fn [S]
                       (go-super S
-                        (throw (ex-info "foo" {}))))]
-       (go (is (thrown? Exception
+                                (throw (ex-info "foo" {}))))]
+       (go (is (thrown? #?(:clj Exception :cljs js/Error)
                         (<? S (restarting-supervisor start-fn :retries 3 :stale-timeout 100))))))
+
+        ;#?(:clj)  ;; TODO: get this one working in cljs
```
Comment (groundedsage) [inline test/superv/async_test.cljc:348]: The CI issue comes from here @TimoKramer @whilo 
Seems this issue was not fixed as the TODO comment is still here.

**Review by whilo — APPROVED** [review]
Nice work, thanks!

**Pull request merged**